packages/terra-alert/docs/MoreInformation.md

```diff
@@ -0,0 +1,150 @@
+- Title: a brief word or phrase that summarizes the notification.  It is rendered in a bold font to gain the attention of the user.  For the standard notification types, if you don't provide a title, then pre-defined text is used.  For a custom alert, the title is optional.
+- Message content: the notification message which can consist of text or HTML.
+- Action area: an optional area to the right of the banner that contains actionable elements (typically buttons) allowing the user to respond to the alert notification.  This area can consist of the following:
+  - Dismiss button: the Alert component can be configured to be dismissible.  If it is to be dismissible, a Dismiss button is rendered in the action area.
+  - Action element: you can configure the Alert component to include additional actionable element(s) in the action area.  typically you would include a button element.  To be consistent with the Dismiss button if it is used, you should use the Terra Button component.
+
+This image shows the layout of the elements of the Alert component.
+![Image](images/alertLayout.png?raw=true)
+
+## Props
+|Prop Name |Type  |Is Required |Default Value |Description |
+|----------|------|------------|--------------|------------|
+|alertAction |element  |optional  |null  |An action element to be added to the action section of the alert.|
+|children  |node or string  |optional  |''  |Child Nodes providing the message content for the alert. Can contain text and HTML.|
+|customIcon  |element  |optional  |null  |The icon to be used for an alert of type custom.|
+|customStatusColor |string  |optional  |''  |The status bar color to be used for an alert of type custom.|
+|onDismiss |function  |optional  |null  |Callback function triggered when Dismiss button is clicked. The presence of this prop will cause the Dismiss button to be included on the alert.|
+|title |string  |optional  |''  |The title for the alert which will be bolded.|
+|type  |string  |optional  |'alert' |The type of alert to be rendered. One of Alert.Opts.Types.ALERT, Alert.Opts.Types.ERROR, Alert.Opts.Types.WARNING, Alert.Opts.Types.ADVISORY, Alert.Opts.Types.INFO, Alert.Opts.Types.SUCCESS, Alert.Opts.Types.GAP_CHECKING, Alert.Opts.Types.OUTSIDE_RECORDS, Alert.Opts.Types.CUSTOM. Use the Alert.Opts.Types attribute of the Alert component for access to these type strings.|
+
+## Examples
+
+### Imports
+When importing the Alert component, it includes the Types object which contains a series of string constants that can be used for setting the 'type' prop for the component as shown in the examples below.
+```jsx
+import React from 'react';
+import Alert from 'terra-alert';
+```
+
+### Examples
+__________
+Basic Alert component of type 'alert' using the default title and plain text for the message content.
```

Comment:
Perhaps we can have a default Alert that doesn't set the type, as you default type to ALERT?